t_sites/LctrmTestSite3/templates/page.html

```diff
@@ -0,0 +1,6 @@
+{% extends "layout.html" %}
+{% block title %}{{ this.title }}{% endblock %}
+{% block body %}
+  <h2>{{ this.title }}</h2>
+  {{ this.body }}
+{% endblock %}
```

Comment:
Do we really need three empty Lektor sites in the repo? 

Reply:
Nope, for tests only, will be deleted